utils_cv/tracking/data.py

```diff
@@ -6,11 +6,14 @@
 
 
 class Urls:
-    base = "https://cvbp.blob.core.windows.net/public/datasets/tracking/"
+    datasets_base = "https://cvbp.blob.core.windows.net/public/datasets/tracking/"
+    models_base = "https://cvbp.blob.core.windows.net/public/models/tracking/"
 
-    cans_path = urljoin(base, "cans.zip")
-    fridge_objects_path = urljoin(base, "odFridgeObjects_FairMOT-Format.zip")
-    carcans_annotations_path = urljoin(base, "carcans_vott-csv-export.zip")
+    cans_path = urljoin(datasets_base, "cans.zip")
+    fridge_objects_path = urljoin(datasets_base, "odFridgeObjects_FairMOT-Format.zip")
+    carcans_annotations_path = urljoin(datasets_base, "carcans_vott-csv-export.zip")
+    mot_challenge_path = urljoin(datasets_base, "MOT17.zip")
```

Comment:
We aren't allowed to host data we don't own ourselves. Can you download from the original non-microsoft location?

Reply:
Thanks - unfortunately I don't think so. I made this change because I tried running the 02 notebook currently in staging and got an error, even using a fresh VM and clone of the repo. The zip that gets downloaded from MOTChallenge website is corrupted for me. I committed the error message here: https://github.com/microsoft/computervision-recipes/blob/c0078e2c5490777ae6e90fc4073d49bfecd28582/scenarios/tracking/02_mot_challenge.ipynb

Do you mind trying a run of 02 in staging just to be sure it's not me?

Reply:
Would this comment also apply to the baseline dla model?

Reply:
I also got the same error like you before with the zip file

Reply:
Also cc'ing @catherine667 and @karineip in case they are able to run the current staging 02 without a bug.

Reply:
@PatrickBue Ah ok, thanks! Here's a suggestion - I can try using a different year from MOTChallenge website and see if it works. What do you think?

Reply:
If that's quick and easy to try then yeah (please @catherine667 let us know if that's not an option).

Generally I wouldn't spend much more time on this since the MAIDAP project officially ended :)

Reply:
No problem, I think it should be really fast to make those edits. Just need to rerun and replace the markdown numbers at the final comparison. I'll give it a try tonight and push updates here. Do you think it's ok to host the baseline FairMOT model in blob for automatic download?

Reply:
Good point. We'd have to check with CELA so better not

Reply:
Sounds good. So I'll:
1. Revert the model download back to the Google drive link. The user will have to manually click and download but I think that's the only completely compliant way. 
2. Make 02 use a different year.